src/android/app/src/main/res/values/strings.xml

```diff
@@ -490,6 +490,14 @@
     <string name="use_black_backgrounds">Black Backgrounds</string>
     <string name="use_black_backgrounds_description">When using the dark theme, apply black backgrounds.</string>
 
+    <!-- Device Orientation -->
+    <string name="device_orientation_title">Screen Orientation</string>
+    <string name="device_orientation_description">How should Lime3DS display your games?</string>
+    <string name="device_orientation_system">Follow System Setting</string>
+    <string name="device_orientation_landscape">Landscape</string>
+    <string name="device_orientation_portrait">Portrait</string>
+    <string name="device_orientation_auto">Follow Device</string>
```

Comment:
As previously mentioned in the original issue, a feature which force-enables auto-rotate is a violation of UX convention. Please remove this option.

Reply:
I don't know why you think it's a violation, especially because it is a setting that the user has to knowingly choose?

Why not keep it in line with the other emulators? But if you really want i can remove it.